### PR TITLE
Alpha on viewer

### PR DESCRIPTION
**Pull request by cjds**
This allows you to have an Alpha for the Three.JS viewer. Simple optional parameter added to Viewer.js

### CODE REVIEW

File: examples/interactivemarkers.html [hunk 1]
```diff
@@ -23,7 +23,8 @@
       divID : 'markers',
       width : 800,
       height : 600,
-      antialias : true
+      antialias : true,
+      alpha:0.5
```
Comment (FuzzAU) [inline examples/interactivemarkers.html]: I don't want to be "that person", but throw some spaces here for consistency :)

Comment (cjds) [inline examples/interactivemarkers.html]: Huh? Sorry, I'm a little confused. Are you talking about consistency of the framework? The parameter is optional. I just put it in an example to show that it is there. I can remove the line if you like.

**Comment by FuzzAU** [comment]
no no, it's good there as an example. I meant 'alpha : 0.5' instead of 'alpha:0.5' 
Just for style consistency. So very minor. Not sure how particular everyone in on this project with style

**Comment by FuzzAU** [comment]
:)

**Comment by cjds** [comment]
:) No, you're right. My brain didn't even see that. Better to err on the side of particularity than not

**Comment by FuzzAU** [comment]
otherwise a good change, simple enough.

**Comment by T045T** [comment]
Looks good, sorry for the huge delay in merging this!

**Pull request merged**